ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/internal/PeerDiscoveryController.java

```diff
@@ -205,6 +205,7 @@ public void start() {
       throw new IllegalStateException("The peer table had already been started");
     }
 
+    LOG.debug("BOOTSTRAP NODES {}", bootstrapNodes.size());
```

Comment:
I think this uppercase log is not necessarily useful IMO

Reply:
removed :-)